src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -55628,16 +55983,17 @@ void M0<T>(T x0, T y0)
         T z0 = x0 ?? y0;
         M1(z0) = x0;
         z0.ToString();
+        M1(z0).ToString();
```

Comment:
>M1(z0).ToString(); [](start = 8, length = 18)

The line above actually changes the flow sate of ```z0``` consider placing this line above it, so that both ```M1(z0)```` calls are based on the same state.